src/app/page.css

```
@@ -1,1002 +1,1164 @@
+  }
+}
+
+/* 스크린 리더 전용 텍스트 */
+.sr-only {
+  position: absolute !important;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.3.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)